src/lightbox/LightboxScreen.js

```diff
@@ -33,6 +33,20 @@ export default class LightboxScreen extends PureComponent<Props> {
 
   render() {
     const { src, message } = this.props.navigation.state.params;
+    // Next I have checked if a normal image component renders with headers passed.
```

Comment:
Just an attempt to check if simple <Image> components can load images with appropriate headers.